public/app/features/manage-dashboards/components/DeletePublicDashboard/DeletePublicDashboardModal.tsx

```diff
@@ -0,0 +1,49 @@
+import { css } from '@emotion/css';
```

Comment:
Maybe we should have a folder for "public dashboards" and then we can put each button in that instead of DeletePublicDashboard fold in components?

Reply:
you mean a PublicDashboard folder inside components folder with all related components? So, Modal, Button and ListPage components?

Reply:
Yep exactly.

Reply:
already done